### PR TITLE
Adding ODC new features and Known Issues

### PR DESCRIPTION
**Pull request by Preeticp**
This PR adds new features and Known Issues for ODC for 4.7
Aligned team label: DevTools
Acked by SMEs and QE

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -170,6 +170,46 @@ See xref:../web_console/creating-quick-start-tutorials.adoc#creating-quick-start
...
+*** Severity-based counts of all vulnerable images,
+*** Count of fixable vulnerabilities,
+*** Number of affected pods for each vulnerable image
+** You can see the severity details of a vulnerability and also launch the Quay user interface, in the context of the manifest of the vulnerable image stored in that repository, to get more details about the vulnerability.
+* After you have the OpenShift Virtualization Operator installed in your cluster, you can create virtual machines by selecting the *Virtual Machines* option on the *+Add* view and then using the templates in the *Developer Catalog*.
+* The web terminal usability is now enhanced:
+** All users can access the web terminal on the console regardless of their privilege level.
+** When the web terminal is inactive for a long period, it stops and provides the user an option to restart it.
+* The pipelines workflow is now enhanced:
+** The pipeline creation process now makes better use of pipelines over the default build config system. Build configs are no longer created by default along with the Pipelines using the *Import from git* flow and the pipeline starts as soon as you create the application.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:195]: s/flow/workflow

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -170,6 +170,46 @@ See xref:../web_console/creating-quick-start-tutorials.adoc#creating-quick-start
...
+** You can now configure pipelines in the *Pipeline builder* page using either the *Pipeline builder* option or the *YAML view* option. You can also use the Operator-installed, reusable snippets and samples to create detailed Pipelines.
+** The *PipelineRun* page now contains a *TaskRuns* tab that lists the associated task runs. You can click on the required task run to see the details of the task run and debug your pipelines.
+** You can now see the following metrics for your pipelines in the *Pipeline Details* page, per pipeline: pipeline run duration, task run duration, number of pipeline runs per day and the pipeline success ratio per day.
+** An *Events* tab is now available on the *Pipeline Run details* and the *Task Run details* pages, which shows the events for a particular PipelineRun or TaskRun.
+* The serverless usability is now enhanced:
+** You can access the *Serving* and *Eventing* pages from the *Administrator* perspective and create serverless components using the console.
+** You can create Camel connectors using the event source creation flow.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:202]: s/flow/workflow

Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:202]: s/flow/workflow

File: release_notes/ocp-4-7-release-notes.adoc (hunk 1 above)
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:195]: s/flow/workflow

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -1161,6 +1201,12 @@ $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1927244[*BZ#1927244*])
 
+* In Openshift Container Platform 4.3 and 4.4, if the user has the console open in multiple tabs, some sidebar links in the *Developer perspective* do not directly link to the project, and there is an unexpected shift in the selected project. This will be resolved in a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1839101[BZ#1839101])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1204]: s/Openshift Container Platform/{product-title}

**Review by vikram-redhat — APPROVED** [review]
lgtm for merging.

**Pull request merged**